proarc-webapp/src/main/java/cz/cas/lib/proarc/webapp/client/widget/mods/oldprint/OldPrintMonographTitleForm.java

```diff
@@ -0,0 +1,191 @@
+﻿/*
+ * Copyright (C) 2017 Jakub Kremlacek
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ */
+package cz.cas.lib.proarc.webapp.client.widget.mods.oldprint;
+
+import cz.cas.lib.proarc.webapp.shared.form.Field;
+import cz.cas.lib.proarc.webapp.shared.form.FieldBuilder;
+import cz.cas.lib.proarc.webapp.shared.form.Form;
+import java.util.List;
+
+/**
+ * Old Print Multipart Monograph based on The NDK Multipart Monograph.
+ *
+ * Cloned and modified from NDKMonographTitle Version 1.1_2
+ * 
+ * @author Jakub Kremlacek
+ *
+ */
+public final class OldPrintMonographTitleForm {
+
+    public Form build() {
```

Comment:
Já bych to držel oddělené. Záměr byl umožnit vytvořit formuláře deklarativně uživatelem. Java form builder je tam víceméně dočasně. Navíc se často ukáže, že je formuláře potřeba mírně odlišit. Vytaženy jsou jen ty nejvíce opakované části, které se skoro neliší mezi formuláři.